static/js/components/FilterCandidateList.jsx

```diff
@@ -117,30 +129,32 @@ const FilterCandidateList = ({
               <FormValidationError message="Invalid date range." />
             )}
             <Controller
-              as={
+              render={({ onChange, value }) => (
                 <KeyboardDateTimePicker
-                  value={formState.startDate}
-                  label="Start (browser local time)"
+                  value={dayjs.utc(value)}
+                  onChange={(e, date) => onChange(dayjs.utc(date))}
+                  label="Start (UTC)"
                   format="YYYY/MM/DD HH:mm"
                   ampm={false}
                   showTodayButton
```

Comment:
The today button still goes to today in browser local time, rather than UTC. E.g.,

Right now its 13:26 PT, if I click on "Today" on the picker, it sets the time to 13:26 UTC, but should be 21:26 UT.

Reply:
Yes, unfortunately the datepicker always uses the browser local time, and there is no callback exposed for changing this. The best we can do is disable to the "Today" button

Reply:
@dannygoldstein that change is pushed